pom.xml

```diff
@@ -71,7 +71,7 @@
         <kafka.version>1.0.0</kafka.version>
         <json.version>20090211</json.version>
         <commonio.version>2.6</commonio.version>
-        <kusto-sdk.version>2.1.0</kusto-sdk.version>
+        <kusto-sdk.version>1.4.2</kusto-sdk.version>
```

Comment:
are we downgrading the SDK version?

Reply:
We were facing issues while performing table access vaildation with the new SDK.
Currently discussing this with Ohad.
We'll be using the latest version of the SDK.
Also, we are verifying these features, will let you know when it's completed and ready to be reviewed.